src/app/app.controller.ts

```diff
@@ -23,4 +28,12 @@ export class AppController {
     healthz(): string {
         return HttpConstants.HEALTHZ_RESPONSE;
     }
+
+    /**
+     * For the uptime statistics report (see Uptime feature brief)
```

Comment:
To @jeffk-kiva's point, I'd love for us to link to a public-facing resource to explain this. Like a Wiki page or something.

Reply:
+1 to @natesulat 

Or if that's too much of a pain, take it out of the comment and make the comment into a quick one-line summary of what this endpoint's purpose is.

Reply:
I agree this comment can go away when we complete the feature.  This PR completes first step of the feature but do not complete feature.   It it fine for now.